sdcm/argus_test_run.py

```diff
@@ -146,27 +146,51 @@ def _prepare_bare_metal_resource_setup(sct_config: SCTConfiguration) -> BaseClou
 
```

Comment:
The main resource of a K8S backend is `cluster`.
It makes no sense to track K8S nodes without cluster, because if you delete nodes in EKS or GKE they will get immediately recreated by the autoscaler.
So, we must refer to a K8S cluster where we will be able to delete `node pools` or scale them to `0` nodes.